packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -49,6 +49,28 @@ import 'typography.dart';
 //
 // 5. If you are a Google employee, you should then also follow the instructions
 //    at go/flutter-l10n. If you're not, don't worry about it.
+//
+// UPDATING AN EXISTING STRING
+//
+// If you (someone contributing to the Flutter framework) want to modify an
+// existing string in the MaterialLocalizations objects, follow these steps:
+//
+// 1. Modify the default value of the relevant getter(s) in
+//    DefaultMaterialLocalizations below.
+//
+// 2. Update the flutter_localizations package. Modify the out-of-date English
+//    strings in lib/src/l10n/material_en.arb.
+//
+//    You also need to re-generate lib/src/l10n/localizations.dart by running:
```

Comment:
The script `dart dev/tools/localization/bin/gen_missing_localizations.dart` had no effect when I ran it at this point, so I'm assuming that it's not necessary to run it.  I've left it out of these docs.